docs/user/dashboard/lens.asciidoc

```diff
@@ -19,301 +25,316 @@ data-type="inline"
 
-Add and define fields to the index pattern that you want to visualize using the {ref}/modules-scripting-painless.html[Painless scripting language].
-The fields that you are add are saved to the index pattern and appear in all visualizations, saved searches, and saved objects that use the index pattern. 
+. Drag the fields you want to visualize to the workspace. 
 
-. Click *...*, then select *Add field to index pattern*.
+. To sort or hide the columns, click the arrow next to the column heading, then select an option.
 +
-[role="screenshot"]
-image:images/runtime-field-menu.png[Dropdown menu located next to index pattern field with items for adding and managing fields, width=50%]
+All columns that belong to the same layer pane group are sorted in the table.
 
-. Enter a *Name* for the field, then select the field *Type*.
+. To change the display options, click a *Metrics* field, then configure the following options:
 
-. Select *Set value*, then define the field value by emitting a single value using the {ref}/modules-scripting-painless.html[Painless scripting language].
+* *Display name* &mdash; Specifies the field display name. 
 
-. Click *Save*.
-+
-To manage the field, click the field, then click *Edit index pattern field* or *Remove index pattern field*.
-+
-[role="screenshot"]
-image:images/manage-runtime-field.gif[Field menu to edit or remove field from index pattern, width=50%]
+* *Value format* &mdash; Specifies how the field value displays in the table. 
 
-For more information about adding fields to index patterns and Painless scripting language examples, refer to <<runtime-fields,Runtime fields>>.
+* *Text alignment* &mdash; Aligns the values in the cell to the *Left*, *Center*, or *Right*.
+
+* *Hide column* &mdash; Hides the column for the field.
+
+* *Summary row* &mdash; Adds a row that displays the summary value. When specified, allows you to enter a *Summary label*.
+
+* *Color by value* &mdash; Applies color to the cell or text values. To change the color, click *Edit*. 
 
 [float]
 [[drag-and-drop-keyboard-navigation]]
-===== Create visualizations with keyboard navigation
+==== Create visualizations with keyboard navigation
 
-*Lens* has a fully accessible, continuously improved drag and drop system, which allows you to use a keyboard instead of a mouse.
+To use a keyboard instead of a mouse, use the *Lens* fully accessible and continuously improved drag system.
 
 [role="screenshot"]
 image::images/lens_drag_drop_1.gif[Presented Lens drag and drop]
 
-. Set a focus on the chosen item. Most of the draggable elements have two focus states. The inner focus state opens a panel with detailed information or options. 
-The outer focus state allows you to drag an item. Tab through the page until you get the outer focus state on the chosen item:
+. Select the field in the fields list or layer pane. Most fields have an inner and outer select state. The inner state opens a panel with detailed information or options. 
```

Comment:
Are you using the word "field" to refer to both `bytes` and `Sum of bytes` sections? That's confusing to me.